compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/platform/DomInputStrategy.kt

```diff
@@ -288,4 +289,11 @@ private class RepeatDetector(private val input: HTMLElement) {
             }
         });
     }
-}
\ No newline at end of file
+}
+
+// TODO: Ideally we should reuse KeyEvent.isTypedEvent
+private val KeyboardEvent.isTypedEvent: Boolean
+    get() = type == "keydown"
+        && !metaKey
+        && !ctrlKey
+        && key.firstOrNull()?.toString() == key
```

Comment:
just to be 100% sure, what do you think about checking `key.length == 1`?

Also, this code uses strings, which would cause strings copying in case of js-k/wasm interop. Moving this logic to js would be better, because this extension will called on almost every keyboard input.